Babylon/groups/azure/groups/acr/commands/ls.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from click import command, make_pass_decorator
+from azure.containerregistry import ContainerRegistryClient
+from Babylon.utils.decorators import requires_external_program, require_deployment_key
+
+logger = logging.getLogger("Babylon")
+
+"""Command Tests
+> babylon acr ls
+Should list all images in the given registry
+Can be checked against `az acr repository list --name my_registry --output table`
+"""
+
+pass_cr_client = make_pass_decorator(ContainerRegistryClient)
+
+@command()
+@requires_external_program("docker")
+@pass_cr_client
+@require_deployment_key("acr_registry_name", "acr_registry_name")
+def ls(cr_client: ContainerRegistryClient, acr_registry_name: str):
```

Comment:
The command does not works :
```text
└▶ babylon azure acr ls
Traceback (most recent call last):
  File "/home/afossart/git_repos/Babylon/.venv/bin/babylon", line 11, in <module>
    load_entry_point('Babylon', 'console_scripts', 'babylon')()
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/afossart/git_repos/Babylon/Babylon/utils/decorators.py", line 150, in wrapper
    func(*args, **kwargs)
  File "/home/afossart/git_repos/Babylon/.venv/lib/python3.9/site-packages/click/decorators.py", line 78, in new_func
    raise RuntimeError(
RuntimeError: Managed to invoke callback without a context object of type 'ContainerRegistryClient' existing.
```

Reply:
I edited the PR friday and did not have the time to fix this one, thanks :+1: